policyengine_us/parameters/gov/hhs/medicaid/eligibility/categories/senior_or_disabled/income/limit/couple.yaml

```diff
@@ -1,113 +1,112 @@
 description: Maximum income limit (of SSI countable income) for a couple to qualify for Medicaid under the optional senior or disabled pathway.
 metadata:
-  unit: currency-USD
-  period: month
+  unit: /1
   label: "Medicaid senior or disabled income limit (couple)"
   breakdown:
   - state_code
   reference:
     - title: "Medicaid Financial Eligibility for Seniors and People with Disabilities: Findings from a 50-State Survey | KFF"
       href: https://www.kff.org/report-section/medicaid-financial-eligibility-for-seniors-and-people-with-disabilities-findings-from-a-50-state-survey-appendix-tables/
 
-AL:
-  2015-01-01: 1_125
 AK:
```

Comment:
Do the values come from Appendix Table 2?
Could you state that in reference in so as there is so much information in the reference

Reply:
![image](https://github.com/user-attachments/assets/7435945b-b406-4d38-a6c1-0b666c726426)

If so, did the values change in 2018? I am seeing different percentages and I dont see e.g. 83% anywhere in this doc